psyneulink/core/llvm/helpers.py

```diff
@@ -571,7 +571,8 @@ def generate_ran_this_trial(self, builder, cond_ptr, node):
         return builder.icmp_signed("==", node_trial, global_trial)
 
     def generate_sched_condition(self, builder, condition, cond_ptr, node,
-                                 is_finished_callbacks, num_exec_locs):
+                                 is_finished_callbacks, num_exec_locs,
+                                 nodes_states):
```

Comment:
using `nodes_states` makes sense. It's a generalization of `num_exec_locs` which can now be removed.
I don't think it should be in this PR, but pls leave a TODO/FIXME or open a bug so it stays on the radar.